src/main/java/com/twilio/oai/api/APIResourceBuilder.java

```diff
@@ -0,0 +1,208 @@
+        this.phpTemplate = template;
+        this.codegenOperationList = codegenOperations;
+        this.allModels = allModels;
+        this.recordKey = findRecordKey();
+    }
+
+    @Override
+    public APIResourceBuilder operations(ISchemaResolver<CodegenParameter> codegenParameterIResolver) {
+        this.codegenOperationList.stream().forEach(codegenOperation -> {
+            codegenOperation.pathParams = codegenOperation.pathParams.stream().map(item ->
+                    codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+            codegenOperation.allParams = codegenOperation.allParams.stream().map(item ->
+                    codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+            codegenOperation.queryParams = codegenOperation.queryParams.stream().map(item ->
+                    codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+            codegenOperation.bodyParams = codegenOperation.bodyParams.stream().map(item ->
+                    codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+            codegenOperation.requiredParams = codegenOperation.requiredParams.stream()
+                    .map(item -> codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+            codegenOperation.optionalParams = codegenOperation.optionalParams.stream()
+                    .map(item -> codegenParameterIResolver.resolve(item)).collect(Collectors.toList());
+
+            requiredPathParams.addAll(codegenOperation.pathParams);
+            codegenOperation.vendorExtensions = mapOperation(codegenOperation);
+        });
+        return this;
+    }
+
+    @Override
+    public APIResourceBuilder responseModel(ISchemaResolver<CodegenProperty> codegenPropertyIResolver) {
+        codegenOperationList.stream().forEach(codegenOperation -> {
+            List<CodegenModel> responseModels = new ArrayList<>();
+            codegenOperation.responses
+                    .stream()
+                    .map(response -> response.dataType)
+                    .filter(Objects::nonNull)
+                    .map(modelName -> this.getModel(modelName, codegenOperation, allModels))
+                    .flatMap(Optional::stream)
+                    .forEach(item -> {
+                        item.vars = item.vars.stream().map(v ->
+                                codegenPropertyIResolver.resolve(v)).collect(Collectors.toList());
+                        item.allVars = item.allVars.stream().map(v ->
+                                codegenPropertyIResolver.resolve(v)).collect(Collectors.toList());
+                        responseModels.add(item);
+                    });
+            this.apiResponseModels = getDistinctResponseModel(responseModels);
+        });
+        return this;
+    }
+
+    @Override
+    public APIResourceBuilder template() {
+        return this;
+    }
```

Comment:
Wonder if this class should be abstract and not implement this method since it does nothing.